src/test/unit/pid_unittest.cc

```diff
@@ -125,7 +134,7 @@ void setDefaultTestSettings(void)
     pidProfile->pid[PID_ROLL]  =  { 40, 40, 30, 65 };
     pidProfile->pid[PID_PITCH] =  { 58, 50, 35, 60 };
     pidProfile->pid[PID_YAW]   =  { 70, 45, 20, 60 };
-    pidProfile->pid[PID_LEVEL] =  { 50, 50, 75, 0 };
+    pidProfile->pid[PID_LEVEL] =  { 50, 50, 75, 100 };
```

Comment:
Please leave as is. See line 120 (129)

Reply:
If we set this to zero then we aren't testing the feedforward calculations properly.